packages/react-lowcode/src/ast/identifierLength.ts

```diff
@@ -7,9 +7,7 @@ import { SourceFile } from "ts-morph"
 
 export const getJsxIdentifierLength = (code: string, source: SourceLineCol) => {
   const pos = startOfJsxIdentifier(code, source)
-  if (pos) {
-    const node = astFindStart(code, pos)
-    if (!node) throw new Error("Unable to find node in AST")
-    return node.end - node.pos
-  }
+  const node = astFindStart(code, pos!)
```

Comment:
never use `!` operator

Reply:
it was just quick fix for me...i dont understand how code which cant be compiled could be pushed to repo